nncf/openvino/quantization/helpers.py

```diff
@@ -96,7 +125,8 @@ def quantize_impl(model: ov.Model,
                 'stat_subset_size': subset_size,
                 'use_fast_bias': fast_error_correction,
                 'model_type': model_type,
-            }
+            },
+            'ignored': _create_ignored_scope_config(ignored_scope)
```

Comment:
some tests are required for sure maybe without running anything but checking the result of `_create_ignored_scope_config` functions

Reply:
Done